res/TensorFlowLiteRecipes/YUV_TO_RGB_U8_000/test.recipe

```diff
@@ -0,0 +1,67 @@
+operand {
+  name: "input"
+  type: FLOAT32
+  shape { dim: 1 dim: 4 dim: 4 dim: 3 }
+}
+operand {
+  name: "weight"
+  type: UINT8
+  shape { dim: 3 dim: 3 }
+  quant { min: 0 max: 1 scale: 0.000553869 zero_point: 0 }
+}
+operand {
+  name: "reshaped"
+  type: FLOAT32
+  shape { dim: 16 dim: 3 }
+}
+operand {
+  name: "outf"
+  type: FLOAT32
+  shape { dim: 16 dim: 3 }
+}
+operand {
+  name: "out"
+  type: FLOAT32
+  shape { dim: 1 dim: 4 dim: 4 dim: 3 }
+}
+operand {
+  name: "bias"
+  type: FLOAT32
+  shape { dim: 3 }
+}
+operation {
+  type: "Reshape"
+  reshape_options {
+    new_shape: 16
+    new_shape: 3
+  }
+  input: "input"
+  output: "reshaped"
+}
+operation {
+  type: "FullyConnected"
+  fullyconnected_options {
+    activation: NONE
+  }
+  input: "reshaped"
+  input: "weight"
+  input: "bias"
+  output: "outf"
+}
+
+operation {
+  type: "Reshape"
+  input: "outf"
+  output: "out"
+  reshape_options {
+    new_shape: 1
+    new_shape: 4
+    new_shape: 4
+    new_shape: 3
+  }
+}
+
+input: "input"
+input: "weight"
+input: "bias"
```

Comment:
I think weight and bias should not be in input of the network

Reply:
See `FullyConnected_001` for reference
https://github.com/Samsung/ONE/blob/9b2a77d442a12fd89bad280d7002013946c1b95d/res/TensorFlowLiteRecipes/FullyConnected_001/test.recipe#L30-L34

Reply:
Um.. there is not strict one but here is my understanding of this recipe.

`FullyConnected_001 ` is to test `FullyConnected_001 ` with inputs weight and bias. For FC, weight and bias can be any thing as it doesn't mater where or what the values are. we are just creating `FullyConnected`

For YUV_to_RGB, our purpose is to test `out = yuv_to_rgb(input)` sematically. there is no weight or bias as input. And `FullyConnected` is happend to be used as operator.

Reply:
@binarman , can you give your opinion about this?

Reply:
@seanshpark 
I agree with you.
At some point we may even try to folds this into circle-specific operator. (for example, for better quantization)
So we need to be more specific in this case.

P.S. @AShedko is absent today and probably tomorrow, so I'll add weights myself